asserts/assertstest/assertstest.go

```diff
@@ -266,7 +273,16 @@ type StoreStack struct {
 }
 
 // NewStoreStack creates a new store assertion stack. It panics on error.
-func NewStoreStack(authorityID string, rootPrivKey, storePrivKey asserts.PrivateKey) *StoreStack {
+// optional privKeys can be in order: root, store, generic
+func NewStoreStack(authorityID string, privKeys ...asserts.PrivateKey) *StoreStack {
```

Comment:
I can check for that, also I went this way because I think lots of tests use explicit keys either to not generate too often or just because, so we could change a few places to not pass any key at all